tensorflow/python/kernel_tests/sparse_ops_test.py

```diff
@@ -743,5 +743,46 @@ def testMismatchedShapes(self):
         tf.sparse_maximum(sp_zero, sp_one).eval()
 
 
+class SparseTransposeTest(tf.test.TestCase):
```

Comment:
I have changed this test now. Now that it is in ``sparse_ops``,  shall I still move it to ``transpose_op_test.py`` now ?